web/src/portico/help.js

```diff
@@ -46,19 +46,33 @@ function add_copy_to_clipboard_element($codehilite) {
         },
     });
 
+    // Show a tippy tooltip when the button is hovered
+    const tooltipCopy = tippy($copy_button[0], {
+        content: "Copy code",
+        trigger: "mouseenter",
+        placement: "top",
+    });
+
     // Show a tippy tooltip when the code is copied
-    clipboard.on("success", () => {
-        const tooltip = tippy($copy_button[0], {
-            content: "Copied!",
-            trigger: "manual",
-            placement: "top",
-        });
+    const tooltipCopied = tippy($copy_button[0], {
+        content: "Copied!",
+        trigger: "manual",
+        placement: "top",
+    });
+
+    // Copy code on button click
+    $copy_button.on("click", () => {
+        clipboard.onClick({currentTarget: $copy_button[0]});
+    });
 
-        tooltip.show();
+    // Show "Copied!" tooltip when code is successfully copied
+    clipboard.on("success", () => {
+        tooltipCopy.hide();
+        tooltipCopied.show();
```

Comment:
Looks weird to look at a mix of CamelCase variable and snake_case. Let's just use snake_case. Another commit to mvoe `registerCodeSection` to snake_case would be welcome as well.